bundles/binding/org.openhab.binding.fritzbox/README.md

```diff
@@ -1,5 +1,11 @@
 # Fritz!Box Binding
 
+## Prerequisites
```

Comment:
I suggest merging this section with the `Prepare your Fritzbox` section below, and put the note about the firmware support below the `Enable Telnet` bullet.

Reply:
It's fine for it to be the first section. But the two sections should be combined, since they both describe prerequisites.

Reply:
Understood now. Please review my changes and resolve if okay.